### PR TITLE
Add warning and link for no installed examples

### PR DESCRIPTION
**Pull request by benmcmorran**
image

### CODE REVIEW

File: src/arduino/exampleProvider.ts [hunk 1]
```diff
@@ -43,10 +43,18 @@ export class ExampleProvider implements vscode.TreeDataProvider<ExampleItem> {
         this._exmaples = null;
```
Comment (gcampbell-msft) [inline src/arduino/exampleProvider.ts:43]: Nit: Mispelling: `_examples` rather than `_exmaples`

File: src/arduino/exampleProvider.ts [hunk 2]
```diff
@@ -43,10 +43,18 @@ export class ExampleProvider implements vscode.TreeDataProvider<ExampleItem> {
         this._exmaples = null;
         this._exampleManager.loadExamples().then((examples) => {
             this._exmaples = examples;
+            if (!this.hasAnyExamples(this._exmaples)) {
```
Comment (gcampbell-msft) [inline src/arduino/exampleProvider.ts:46]: If there aren't any examples, won't this array already be empty?

Comment (gcampbell-msft) [reply src/arduino/exampleProvider.ts:46]: Oh, I see it's more complex of a check than that, ignore.

**Review by gcampbell-msft — APPROVED** [review]

File: src/arduino/exampleProvider.ts (hunk 2 above)
Comment (benmcmorran) [reply src/arduino/exampleProvider.ts:46]: Yep, turns out that the "Built-in examples" node is always inserted, so this check if verifying that there are actually leaves in the tree (which represent real examples).

**Pull request merged**